core/quantity/src/Formatter/Format.ts

```diff
@@ -415,6 +470,13 @@ export class Format {
     }
   }
 
+  /** Create a Format from FormatProps */
+  public static async createFromJSON(name: string, unitsProvider: UnitsProvider, formatProps: FormatProps) {
```

Comment:
Not sure I seee the reason for this method instead of just letting people use the 3 calls you have within it. 

We could alternatively have a constructor that takes the JSON and does the same thing the fromJSON does.

Reply:
This is a convenienc method, so I could replace a bunch of caller from always calling two methods back to back. It can't be done in constructor because it loads units from UnitsProvider and that is async.